docs/operations/security-overview.md

```diff
@@ -264,3 +318,29 @@ As an alternative to using the basic metadata authenticator, as shown in the pre
 
 
 Congratulations, you have configured permissions for user-assigned roles in Druid!
+
+
+## Druid security trust model
+Like all other security systems, trust is the foundation of the Druid security model. Druid administrators and read-only users are trusted users. Therefore, they are not expected to act maliciously.
```

Comment:
I think it's reasonable to expect administrators to be trusted users, but the requirement that read-only act in a non-malicious way is not necessarily true.

Reply:
Actually, I think any authenticated user is expected to not act in a non-malicious way. If you are worried about those users, you should build another layer on top of Druid?

Reply:
What can a read-only user do to attack? Asking for my own knowledge. Like if I can't write something, I can't submit malicious code or other, right? (I may be naive on this front).